tests/product/prestoadmin_installer.py

```diff
@@ -51,14 +52,11 @@ def install(self, cluster=None, dist_dir=None):
         if not dist_dir:
             dist_dir = self._build_dist_if_necessary(cluster)
         self._copy_dist_to_host(cluster, dist_dir, cluster.master)
-        cluster.copy_to_host(
-            LOCAL_RESOURCES_DIR + "/install-admin.sh", cluster.master)
-        cluster.exec_cmd_on_host(
-            cluster.master,
-            'chmod +x ' + cluster.mount_dir + "/install-admin.sh"
-        )
-        cluster.exec_cmd_on_host(
-            cluster.master, cluster.mount_dir + "/install-admin.sh")
+        with open(LOCAL_RESOURCES_DIR + "/install-admin.sh", 'r') as file_obj:
+            script = ''.join(file_obj.readlines())
```

Comment:
I think that's less convenient. The whole code block would look like this:

```
file_obj = open(path)
script = file_obj.read(hardcoded_size)
file_obj.close()
```

The `with` context manager closes the file for me and I don't have to hardcode a size with the current version.